src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public class Ball {
+    private int number;
+    private String color;
+
+    public Ball(int number, Color color) {
+        this.number = number;
+        this.color = color.name();
```

Comment:
passing `Color` object makes no sense - you can pass `String` directly

apart from that, by doing `color.name`, you'll get the collor in upper case (`GREEN` for example, although you need `green`)

```suggestion
    public Ball(int number, String color) {
        this.number = number;
        this.color = color;
```